### ISSUE TITLE
Controls and tabs duplicate when ESP is restarted

### ISSUE THREAD

**Issue by enwi** [s00500/ESPUI#83]
**Describe the bug**
The controls and tabs on the web page duplicate, when the ESP is restarted and the browser tab still opened.

**To Reproduce**

1. Start the ESP with example code
2. Open the webpage
3. Turn off the ESP
4. Wait until the webpage displays `Error / No Connection ↻`
5. Turn on the ESP
6. After the webpage reconnects to the ESP all controls and tabs are duplicated

**Expected behavior**
Controls and tabs should not be duplicated after a restart of the ESP ('backend')

**Screenshots**
After step 4:
<img width="1154" alt="Bildschirmfoto 2020-06-21 um 15 54 45" src="https://user-images.githubusercontent.com/25170066/85226523-39aa6e80-b3d8-11ea-8f97-75037e3085d4.png">

After step 6 (picture scaled to fit everything on it):
<img width="1193" alt="Bildschirmfoto 2020-06-21 um 15 57 53" src="https://user-images.githubusercontent.com/25170066/85226532-43cc6d00-b3d8-11ea-81f8-46ba17d75e76.png">

**Desktop (please complete the following information):**
 - OS: iOS
 - Browser: Cliqz
 - Version 1.36.0 (76.0.1+1.46.0.168)(64-Bit)

**Additional context**
ESP used: ESP8266

**Comment by DeeJayMX** [comment]
Same here with chrome

**Comment by s00500** [comment]
Hehe I think I know why this is happening... thanks will fix

**Comment by s00500** [comment]
Should be fixed here: https://github.com/s00500/ESPUI/tree/add_method_for_remove_controls

**Comment by s00500** [comment]
This has been merged into 2.0.1
